backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/ConfirmationCodesController.java

```diff
@@ -0,0 +1,69 @@
+package ca.gov.dtsstn.cdcp.api.web.v1.controller;
+
+import org.mapstruct.factory.Mappers;
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.http.HttpStatus;
+import org.springframework.util.Assert;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.gov.dtsstn.cdcp.api.config.SpringDocConfig.OAuthSecurityRequirement;
+import ca.gov.dtsstn.cdcp.api.service.UserService;
+import ca.gov.dtsstn.cdcp.api.service.domain.User;
+import ca.gov.dtsstn.cdcp.api.web.exception.ResourceNotFoundException;
+import ca.gov.dtsstn.cdcp.api.web.v1.model.ConfirmationCodeModel;
+import ca.gov.dtsstn.cdcp.api.web.v1.model.mapper.ConfirmationCodeModelMapper;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.constraints.NotBlank;
+
+
+@Validated
+@RestController
+@OAuthSecurityRequirement
+@RequestMapping({ "/api/v1/users/{userId}/confirmation-codes" })
+@Tag(name = "Confirmation codes", description = "Endpoint for managing user email confirmation codes.")
```

Comment:
**Nitpick:** Since this controller doesn't deal with emails.

```suggestion
@Tag(name = "Confirmation codes", description = "Endpoint for managing confirmation code resources.")
```

Reply:
fixed